packages/types/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@scalar/types",
+  "description": "Types to work with Scalar packages",
+  "license": "MIT",
+  "author": "Scalar (https://github.com/scalar)",
+  "homepage": "https://github.com/scalar/scalar",
+  "bugs": "https://github.com/scalar/scalar/issues/new/choose",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/scalar/scalar.git",
+    "directory": "packages/types"
+  },
+  "keywords": [
+    "typescript",
+    "types",
+    "scalar"
+  ],
+  "version": "0.0.1",
+  "engines": {
+    "node": ">=18"
+  },
+  "scripts": {
+    "build": "tsc",
+    "lint:check": "eslint .",
+    "lint:fix": "eslint .  --fix"
+  },
+  "type": "module",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "CHANGELOG.md"
+  ],
+  "dependencies": {
+    "@scalar/openapi-parser": "^0.7.2",
```

Comment:
I think this eliminates the whole point of the package? :joy:

Reply:
Not really! The @scalar/openapi-parser dependency tree is way smaller, also I’ve already submitted a PR to separate the types into a dependency free `@scalar/openapi-types`, so we can use that very soon.

Reply:
Wait a sec how did this get here lol

Reply:
Oh looks like hans made some changes. I was like what haha

Reply:
@hanspagel these are the dependencies from the openapi parser
```bash
ajv 8.17.1
├── fast-deep-equal 3.1.3
├── fast-uri 3.0.1
├── json-schema-traverse 1.0.0
└── require-from-string 2.0.2
ajv-draft-04 1.0.0
└─┬ ajv 8.17.1 peer
  ├── fast-deep-equal 3.1.3
  ├── fast-uri 3.0.1
  ├── json-schema-traverse 1.0.0
  └── require-from-string 2.0.2
ajv-formats 3.0.1
└─┬ ajv 8.17.1 peer
  ├── fast-deep-equal 3.1.3
  ├── fast-uri 3.0.1
  ├── json-schema-traverse 1.0.0
  └── require-from-string 2.0.2
jsonpointer 5.0.1
leven 4.0.0
yaml 2.4.5
```
ajv, ajv-draft-04, ajv-draft-04 and jsonpointer  all cause trouble due to them being commonjs
yaml causes problems in other ways

We need that new types package